src/hl.h

```diff
@@ -707,6 +707,7 @@ typedef struct {
 HL_API void *hl_fatal_error( const char *msg, const char *file, int line );
 HL_API void hl_fatal_fmt( const char *file, int line, const char *fmt, ...);
 HL_API void hl_sys_init(void **args, int nargs, void *hlfile);
+HL_API double hl_sys_time();

```

Comment:
I prefer this to be declared as extern in alloc.c than made a public "official api" function in hl.h